dialpad/client.py

```diff
@@ -34,10 +34,23 @@
 
 
 class DialpadClient(object):
-  def __init__(self, token, sandbox=False, base_url=None):
+  def __init__(self, token, sandbox=False, base_url=None, company_id=None):
     self._token = token
     self._session = requests.Session()
     self._base_url = base_url or hosts.get('sandbox' if sandbox else 'live')
+    self._company_id = company_id
+
+  @property
+  def company_id(self):
+    return self._company_id
+
+  @company_id.setter
+  def company_id(self, value):
+    self._company_id = value
+
+  @company_id.deleter
+  def company_id(self):
+    del self._company_id
```

Comment:
Should this be self._company_id = None?

Reply:
@jakedialpad I was in doubt actually. This would delete the property all together, next time we want to access it, it raises an error. The official python guide suggest this for property deleter. I believe they assume setting to None should be done through setter.